ui/app/AppLayouts/Profile/views/wallet/AccountView.qml

```diff
@@ -305,8 +305,11 @@ ColumnLayout {
 
     Component {
         id: renameAccountModalComponent
-        RenameAccontModal {
-            account: root.account
+        RenameAccountModal {
+            accountName: root.account?.name
```

Comment:
In this case using optional chaining doesn't seem to be fully correct. It may return `undefined` and it will break the binding and print a warning (like `unable to assign [undefined] to "accountName"`). The safe option should return `null` in case of failure instead of `undefined`, e.g.:

```
accountName: root.account?.name ?? null
```